ezomero/_importer.py

```diff
@@ -468,6 +470,7 @@ def ezimport_ln_s(self) -> bool:
                     '-k', self.conn.getSession().getUuid().val,
                     '-s', self.conn.host,
                     '-p', str(self.conn.port),
+                    '--depth', '10',
```

Comment:
I'm going to suggest that, instead of hardcoding that, `ezimport` can take extra arguments that get passed through to the `import` call at CLI level. This way, if we ever need `--depth 20`, or if folks need to pass any other optional import arguments, that doesn't require any code change.

Reply:
@erickmartins thanks for the feedback. Do you mean adding it as a keyword argument to the `Importer` class?

Reply:
Thanks for working on this! I'm on parental leave until early April so thanks big-time for the fix :)